.github/workflows/conan.yml

```diff
@@ -0,0 +1,28 @@
+name: conan-fastnoise-lite CI
+
+on:
+  push:
+    branches: [ ci-test ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+   linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - name: Install dependencies
+      run: sudo pip install conan
+    - name: Config
+      env:
+        CONAN_LOGIN_USERNAME: ${{ secrets.JFROG_USER }}
+        CONAN_PASSWORD: ${{ secrets.JFROG_APIKEY }}
+      run: |
+        conan remote add triada https://triada.jfrog.io/artifactory/api/conan/conan
+        conan user -r triada -p --skip-auth
```

Comment:
how does auth work here?

Reply:
It causes conan to skip interactive authentication process and uses `CONAN_LOGIN_USERNAME` and `CONAN_PASSWORD` for authentication later